app/assets/stylesheets/sass/views/attendances/_new.css.scss

```diff
@@ -36,14 +34,15 @@ body {
           width: 25rem;
           
           div.screen {
```

Comment:
I recommend using only the name of the classes to write the styles of the elements:
I.e. "div.screen" --> ".screen"
And like this for all the elements 😄